modules/nw-route-set-or-delete-http-headers.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * networking/route-configuration.adoc
+
+:_content-type: PROCEDURE
+[id="nw-route-set-or-delete-http-headers_{context}"]
+= Setting or deleting HTTP request and response headers in a route
+
+Certain HTTP request and response headers might need to be set or deleted for compliance purposes or other reasons. You can set or delete these headers either for all routes served by an Ingress Controller or for specific routes.
+
+For example, you might want to enable a web application to serve content in alternate locations for specific routes if that content is written in multiple languages, even if there is a default global location specified by the Ingress Controller serving the routes.
+
+The following procedure creates a route that sets the Content-Location HTTP request header so that the URL associated with the application, `\https://app.example.com`, directs to the location `\https://app.example.com/lang/en-us` that way anyone using that specific route is accessing web content written in American English.
+
+.Prerequisites
+. You have installed the OpenShift CLI (`oc`).
+. You are logged into an {product-title} cluster as a project administrator.
+. You have an web application that exposes a port and an HTTP or TLS endpoint listening for traffic on the port.
+
+.Procedure
+. Create a route definition, `app-example-route.yaml`:
```

Comment:
This should be expanded some. Could write something like:

* Create a route definition and save it in a file called `app-example-route.yaml`:
* You could leave the file name out of the step and put it in as the title of the code block. An example of this is here: https://docs.openshift.com/container-platform/4.14/nodes/pods/nodes-pods-secrets-store.html#secrets-store-aws_nodes-pods-secrets-store

I'm sure there are other examples in the repo you could find. It just looks like an incomplete sentence as is.